README.md

```diff
@@ -17,7 +17,9 @@ The Teleporter protocol, on the other hand, is implemented at the smart contract
 - [E2E tests](#e2e-tests)
   - [Run specific E2E tests](#run-specific-e2e-tests)
   - [Run the E2E tests on another network](#run-the-e2e-tests-on-another-network)
-- [Deploy Teleporter to a Subnet](#deploy-teleporter-to-a-subnet)
+- [Upgradeability](#upgradeability)
+- [Deploy Teleporter to a subnet's chain](#deploy-teleporter-to-a-subnets-chain)
+- [Deploy TeleporterRegistry to a subnet's chain](#deploy-teleporterregistry-to-a-subnets-chain)
```

Comment:
```suggestion
- [Deploy TeleporterMessenger to a Subnet](#deploy-teleporter-to-a-subnets-chain)
- [Deploy TeleporterRegistry to a Subnet](#deploy-teleporterregistry-to-a-subnets-chain)
```

Nit: "Subnet" should be treated as a proper noun and capitalized anywhere it's used in sentences to keep consistent with our broader documentation.

Reply:
does it matter to specify "blockchain" instead of "Subnet?

Reply:
For the sake of the section titles here, I don't think so personally, but I do appreciate the the precision between the two normally haha